packages/video_player_avplay/lib/video_player.dart

```diff
@@ -712,6 +712,17 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     return _videoPlayerPlatform.getStreamingProperty(_playerId, type);
   }
 
+  /// Sets specific feature values for HTTP, MMS, or specific streaming engine (Smooth Streaming, HLS, DASH, DivX Plus Streaming, or Widevine).
+  /// The available streaming properties depend on the streaming protocol or engine.
+  /// Use the CUSTOM_MESSAGE property for streaming engine or CP-specific settings.
```

Comment:
Just curious, I see [this description](https://developer.samsung.com/smarttv/develop/api-references/samsung-product-api-references/avplay-api.html?device=htv) is in the avplay document, but is this actually a feature we can support?

Reply:
The Web avplay plugin and flutter avplay  plugin both based on plusplayer, if Web avplay plugin can support, we can also support it.